### DISCUSSION TITLE
Welcome to openwifi Discussions!

### DISCUSSION THREAD

**Discussion by JiaoXianjun** [open-sdr/openwifi#178] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danyvak** [comment]
Hi 
I'm electronics engineer, interested recently in RF communications and protocols.
I was wondering, if I implement DSSS modem in parallel to openofdm, and connect it to the mac layer, will it provide me a 802.11b compatibility? I mean can lower mac handle a management of the 802.11b messages, once despreded and decoded?
Regards
Dany Vak

**Reply by JiaoXianjun** [reply]
I haven't looked into the detail, but in principle it should work (of course some tuning might be needed).
Because the interface (content wise) between PHY and LMAC is the bytes of MPDU. If DSSS has the same MPDU format, it should work.

You need to consider the PHY info (control wise) exchange between LMAC and DSSS PHY, such as the rate and length field, etc.

**Comment by danyvak** [comment]
Hi 
I have another general question regarding a project: did someone tried to migrate to the newer Vivado version, probably windows?
Or most of the researcher keep working with the 2018.3 version?
Regards
Dany Vak
